File: platform/script-debugger/protocol/protocol-reader/src/org/jetbrains/protocolReader/JsonProtocolModelParseException.java
// Copyright (c) 2009 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

package org.jetbrains.protocolReader;

/**
 * Signals that JSON model has some problem in it.
 */
public class JsonProtocolModelParseException extends RuntimeException {

  public JsonProtocolModelParseException(String message, Throwable cause) {
    super(message, cause);
  }

  public JsonProtocolModelParseException(String message) {
    super(message);
  }
}

File: platform/script-debugger/protocol/protocol-reader/src/JsonProtocolModelParseException.kt
package org.jetbrains.protocolReader

/**
 * Signals that JSON model has some problem in it.
 */
fun JsonProtocolModelParseException(message: String): JsonProtocolModelParseException = JsonProtocolModelParseException(message, null)

class JsonProtocolModelParseException(message: String, cause: Throwable?) : RuntimeException(message, cause)
